### DISCUSSION TITLE
What characteristic is appropriate for digital token as a method of payment?

### DISCUSSION THREAD

**Discussion by tktkowash** [JasmyCoin/JasmyCoin#22] [General]
What characteristic is appropriate for digital token as a method of payment?

**Comment by harayamada** [comment]
If it is to be used as a means of payment to enjoy a certain benefit（products,servicies) in the real economy, it should be of a character that can be used anytime, anywhere, and with anyone.  and be trustless。

It is appropriate to have a means of payment whose value also remains constant,unchangable.
Because you will hesitate to pay at that time、if it will be cheaper next week than tomorrow.

In addition,  it is not the token that has value, but the product。
And if we assume that the value is the product, not the token, we can compare the value of the product by token。

(※So, you want to increase the value of products, not token！
So, how do we maximize the demand for products that wanted, wanted to obtain, and to enjoy the service?
I think it is more important to maximize the demand for the product. this is most important.
tokens are not a commodity. ）

**Comment by tktkowash** [comment]
Focusing on stable or not stable, I think stable would be better. We should consider "loss aversion", well-known human tendency in behavioral economics. If digital token is not stable and has high volatility, people would avoid to use it because of loss aversion. It isn't the matter whether value of digital token would goes up or down.

**Reply by harayamada** [reply]
Loss aversion is a great point for stablecoins. In general, risk aversion has to be considered when it comes to volatility of cryptocurrency and tokens, especially for businesses.   A business has to report their income and do tax filing based on fiat currency values, anyway. So holding and trading assets will just negatively influence the predictability of the business. You can only avoid that by not holding crypto in your accounts, but immediately exchanging it again on an exchange.   Due to the volatility, the only reason for holding onto crypto is speculative investment, which most businesses try to avoid.

**Comment by tktkowash** [comment]
Trustworthiness might indeed be an important point for this discussion.  Cryptocurrency like Bitcoin argues with the fact that it is NOT controlled by a central bank, which actually creates trustworthiness.    In America, the sentiment towards the federal reserve is worsening because of the lax monetary easing policies employed in the country right now. This means, the central bank is printing a lot of money due to the economic situation, and people are afraid of hyperinflation.

Stablecoins are directly pegged to fiat money, either via undelying asset held in contempt for the issuance of the stablecoin, or now the central banks also adopt issuing currency directly in the form of Central Bank Digital Currency (CBDC).
Which from the fiscal dynamics point of view should be seen and treated in the same way as fiat money itself.

**Comment by harayamada** [comment]
OK!
If we think of digital coins as a means of payment in the real economy, without any speculative intentions or as a tool for fundraising, the point is that they need to be used smoothly with anyone, anywhere in the world in a stress-free and trustless environment.
The point is that it needs to be able to be used smoothly without any loss of value. I think we are all in agreement on this.

In order to achieve this, it would be better to have a "stablecoin" (a coin whose value is supported by or based on something solid that the user believes in) that does not rise and fall in value as much as possible, just like legal tender, which is established in the current divided economic sphere within national borders.
The mainstream of this trend is simply the digitization of legal tender, using the U.S. dollar as a base for its value.
On the other hand, if you look up, you can see the movement of CBDCs such as the digital 元. Even if the U.S. dollar is digitized, I think the trend to peg it to the digital dollar will not change.

**Reply by harayamada** [reply]
If we assume that the merit of digital coins is that they can be used more easily by people around the world across national borders, what kind of digital currency with a character not found in legal tender would be better? FOR expanding the economic sphere of the real economy? The fundamental question is.

**Reply by harayamada** [reply]
I am not trying to deny the existence of legal tender／fiat！

The total number of points issued by Rakuten Points is over 2 trillion. From this fact
If the ideal digital currency is one that can be used relatively by anyone anywhere in the world as a means of payment
It may be necessary to think about the point where it should be superior to fiat.

**Reply by harayamada** [reply]
One possible point  is the speed of currency circulation.
If we assume that  larger 経済圏 means more transactions for the exchange of goods, then the number of times a currency is used for transactions in a given period of time should also increase.
The question is how to make speed of digital coins circulation faster than fiat (Of course, it has to do with the amount of currency.）

**Reply by tktkowash** [reply]
If you make a token for only fundraising, then it will probably be categorized as a security -> securities law applies.
The idea of utility tokens is exactly to use these tokens for the business economy, so basically some form of settlement.

**Reply by kmollenkamp** [reply]
Well said.
